### PR TITLE
Fix RAG prompt to pass top hit as context

### PR DESCRIPTION
**Pull request by leemthompo**
Flagged by @lio-p 🙏

### CODE REVIEW

File: notebooks/integrations/openai/openai-KNN-RAG.ipynb [hunk 1]
```diff
@@ -353,16 +353,15 @@
...
+    "pretty_response(response)\n",
+    "top_hit_summary = response['hits']['hits'][0]['_source']['text'] # Store content of top hit for final step"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "276c1147",
    "metadata": {},
    "source": [
-    "## Next steps\n",
```
Comment (leemthompo) [inline notebooks/integrations/openai/openai-KNN-RAG.ipynb:364]: This is just a drive-by 🚗  markdown clean up

**Review by joemcelroy — APPROVED** [review]
